README.md

```diff
@@ -269,6 +274,7 @@ helm install alfresco-incubator/alfresco-dbp \
 --set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://localhost-k8s/auth/" \
 --set alfresco-infrastructure.alfresco-activiti-cloud-registry.enabled=false \
 --set alfresco-infrastructure.alfresco-api-gateway.enabled=false \
+--set alfresco-content-services.externalProtocol="https" \
```

Comment:
I don't think we want self-signed https as the default locally do we?  Won't there be all kinds of browser warnings the developer will have to work around?

Reply:
The cert is already installed by default.

Reply:
Updated to follow how noted in AWS steps